weave/tests/test_weaveflow.py

```diff
@@ -256,14 +256,15 @@ def get_item(row):
     assert res == {"in": 1, "out": 1}
 
 
-def test_op_method_name(client):
-    class MyModel(weave.Model):
-        a: int
+class MyModel(weave.Model):
```

Comment:
is this just style or is there an issue with defining the class inside the func?